test/Kestrel.FunctionalTests/Http2Tests.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2;
+using Microsoft.AspNetCore.Testing;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
+{
+    public class Http2Tests
+    {
+        [Fact]
+        public async Task ServesHttp2WithPriorKnowledge()
```

Comment:
No ALPN test? or do you not have an ALPN client?

Reply:
Currently working on more tests. Not sure I'll be able to final an ALPN client.